solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -788,15 +879,16 @@ describe("RandomBeacon - Relay", () => {
         }
       )
 
-      context("when token staking seize call fails", async () => {
-        let tokenStakingFake: FakeContract<IRandomBeaconStaking>
+      // FIXME: Blocked by https://github.com/defi-wonderland/smock/issues/101
```

Comment:
I am not entirely following, why does it work in `RandomBeacon.GroupCreation.ts` and does not work here?

Reply:
Yeah, my bad. Resolving.